cmake/Gems.cmake

```diff
@@ -76,6 +92,13 @@ macro(o3de_gem_setup default_gem_name)
     o3de_read_json_key(gem_version ${gem_json} "version")
 
     o3de_restricted_path(${gem_json} gem_restricted_path gem_parent_relative_path)
+    if(gem_restricted_path)
+        message(STATUS "Restricted found in ${gem_json}, using: ${gem_restricted_path}")
+    else()
+        # If the gem.json does not have a 'restricted' field use the gem dir and append /restricted
+        set(gem_restricted_path "${gem_path}/restricted")
+        message(STATUS "No restricted found in ${gem_json}, using default: ${gem_restricted_path}")
```

Comment:
same question here, `STATUS` ? If we switch to `VERBOSE` it can be enabled only if you want it, using /V